src/errors/Messages.js

```diff
@@ -32,8 +32,7 @@ const Messages = {
   COLOR_RANGE: 'Color must be within the range 0 - 16777215 (0xFFFFFF).',
   COLOR_CONVERT: 'Unable to convert color to a number.',
 
-  EMBED_FIELD_NAME: 'MessageEmbed field names may not be empty.',
-  EMBED_FIELD_VALUE: 'MessageEmbed field values may not be empty.',
+  EMBED_INVALID_TYPE: (location, must) => `MessageEmbed ${location} must be ${must}.`,
```

Comment:
```suggestion
  EMBED_INVALID_TYPE: (location, must, an = false) => `MessageEmbed ${location} must be a${an ? 'n' : ''} ${must}.`,
```
Missing the article here

Reply:
considered an article, don't think it adds anything of use